justfile

```diff
@@ -127,3 +127,16 @@ ready:
   just test
   just test-doc
   git diff --exit-code --quiet
+
+# Creates a new crate
+new-crate name:
+  cargo new --lib crates/{{snakecase(name)}}
+  cargo run -p xtask_codegen -- new-crate --name={{snakecase(name)}}
+
+# Creates a new changeset for the final changelog. ONLY FOR CRATES, FOR NOW
+new-changeset:
+    knope document-change
+
+dry-run:
```

Comment:
`dry-run` seems too generic to me... Maybe `dry-release`?

Reply:
I'll change it once we have something that works. For now, I am keeping it here as a debugging tool. The name isn't set on stone